### PR TITLE
[ID] Add `Play Style/Hybrid`

### PR DESCRIPTION
**Pull request by Fhoxydeo**

### CODE REVIEW

File: wiki/Play_style/Hybrid/id.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  - keyboard
+  - tap
+  - hybrid
+  - play style
+  -gaya bermain
+---
+
+# Hybrid
+
+**Hybrid** adalah gaya tapping untuk [osu!](/wiki/Game_mode/osu!) yang di mana pemain menekan circle, menggeser slider, dan memutar spinner dengan menggunakan kombinasi dari [*single-tapping*](/wiki/Play_style/Single-tapping) dan/atau [*alternating*](/wiki/Play_style/Alternating). gaya ini dapat ditemukan dari semua pemain tingkat kemahiran dan keterampilan yang dibutuhkan untuk menggunakannya dapat bervariasi dari beatmap ke beatmap dan variasi ke variasi.
```
Comment (wowcake) [inline wiki/Play_style/Hybrid/id.md:12]:
```suggestion
**Hybrid** adalah gaya tapping untuk [osu!](/wiki/Game_mode/osu!) yang di mana pemain menekan circle, menggeser slider, dan memutar spinner dengan menggunakan kombinasi dari [*single-tapping*](/wiki/Play_style/Single-tapping) dan [*alternating*](/wiki/Play_style/Alternating). gaya ini dapat ditemukan dari semua pemain tingkat kemahiran dan keterampilan yang dibutuhkan untuk menggunakannya dapat bervariasi dari berbagai beatmap dan variasi ke variasi.
```

Comment (wowcake) [reply wiki/Play_style/Hybrid/id.md:12]: since it's a combination, you should remove the "/atau"

File: wiki/Play_style/Hybrid/id.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+## Penggunaan/teknik
+
+*Hybrid* dapat dilakukan dengan berbagai cara:
+
+- Beberapa pemain hybrid menekan [*hit object*](/wiki/Hit_object) dengan menekan antara tombol mereka single-tap (`Z` dan `X` secara default) secara berkala di seluruh beatmap. Sementara juga alternating pada [*stacks*](/wiki/Mapping_Techniques/Stack) bila memungkinkan.
```
Comment (wowcake) [inline wiki/Play_style/Hybrid/id.md:18]:
```suggestion
- Beberapa pemain hybrid menekan [*hit object*](/wiki/Hit_object) dengan menekan antara tombol mereka single-tap (`Z` dan `X` secara default) secara berkala di seluruh beatmap. Sementara juga alternating pada [*stack*](/wiki/Mapping_Techniques/Stack) bila memungkinkan.
```

Comment (wowcake) [reply wiki/Play_style/Hybrid/id.md:18]: remember to remove pronoun if there's one in an untranslated/original term @_@

**Review by wowcake — CHANGES_REQUESTED** [review]

File: wiki/Play_style/Hybrid/id.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
+---
+tags:
+  - keyboard
+  - tap
+  - hybrid
+  - play style
+  -gaya bermain
```
Comment (ZeroPyrozen) [inline wiki/Play_style/Hybrid/id.md:7]:
```suggestion
  - gaya bermain
```

File: wiki/Play_style/Hybrid/id.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+---
+
+# Hybrid
+
+**Hybrid** adalah gaya tapping untuk [osu!](/wiki/Game_mode/osu!) yang di mana pemain menekan circle, menggeser slider, dan memutar spinner dengan menggunakan kombinasi dari [*single-tapping*](/wiki/Play_style/Single-tapping) dan [*alternating*](/wiki/Play_style/Alternating). gaya ini dapat ditemukan dari semua pemain tingkat kemahiran dan keterampilan yang dibutuhkan untuk menggunakannya dapat bervariasi dari berbagai beatmap dan variasi ke variasi.
```
Comment (ZeroPyrozen) [inline wiki/Play_style/Hybrid/id.md:12]:
```suggestion
**Hybrid** adalah gaya tapping untuk [osu!](/wiki/Game_mode/osu!) yang dimana pemain menekan circle, menggeser slider, dan memutar spinner dengan menggunakan kombinasi dari [*single-tapping*](/wiki/Play_style/Single-tapping) dan [*alternating*](/wiki/Play_style/Alternating). Gaya ini dapat ditemukan dari semua pemain tingkat kemahiran dan keterampilan yang dibutuhkan untuk menggunakannya dapat bervariasi dari berbagai beatmap dan variasi ke variasi.
```

File: wiki/Play_style/Hybrid/id.md (hunk 3 above)
Comment (ZeroPyrozen) [inline wiki/Play_style/Hybrid/id.md:7]:
```suggestion
  - gaya bermain
```

File: wiki/Play_style/Hybrid/id.md [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+
+## Penggunaan/teknik
+
+*Hybrid* dapat dilakukan dengan berbagai cara:
+
+- Beberapa pemain hybrid menekan [*hit object*](/wiki/Hit_object) dengan menekan antara tombol mereka single-tap (`Z` dan `X` secara default) secara berkala di seluruh beatmap. Sementara juga alternating pada [*stack*](/wiki/Mapping_Techniques/Stack) bila memungkinkan.
```
Comment (ZeroPyrozen) [inline wiki/Play_style/Hybrid/id.md:18]:
```suggestion
- Beberapa pemain hybrid menekan [*hit object*](/wiki/Hit_object) dengan menekan antara tombol mereka single-tap (`Z` dan `X` secara default) secara berkala pada beatmap. Sementara juga alternating pada [*stack*](/wiki/Mapping_Techniques/Stack) bila memungkinkan.
```

File: wiki/Play_style/Hybrid/id.md [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+- Beberapa pemain hybrid menggunakan kombinasi dari alternating dan single-tapping tergantung pada kecepatan sebuah beatmap: beberapa kecepatan lebih mudah untuk detekan dengan singletapping daripada alternating dan banyak pemain hybrid mengakomodasi untuk itu. Kecepatan di mana ini bervariasi dari pemain ke pemain.
```
Comment (ZeroPyrozen) [inline wiki/Play_style/Hybrid/id.md:19]:
```suggestion
- Beberapa pemain hybrid menggunakan kombinasi dari alternating dan single-tapping tergantung pada kecepatan sebuah beatmap: beberapa kecepatan lebih mudah untuk detekan dengan singletapping daripada alternating dan banyak pemain hybrid mengakomodasi untuk itu. Kecepatan dimana ini bervariasi dari pemain ke pemain.
```

**Review by ZeroPyrozen — CHANGES_REQUESTED** [review]

**Review by ZeroPyrozen — APPROVED** [review]
All good 👍

**Review by wowcake — APPROVED** [review]
👍

**Review by TPGPL — APPROVED** [review]

**Pull request merged**